prisma/schema.prisma

```diff
@@ -11,14 +11,110 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Log {
-  id      String @id @default(uuid())
-  level   Level
-  message String
+model Ship {
+  id        String   @id @default(uuid())
```

Comment:
Am I going overboard on UUIDs? Maybe? Am I traumatized by XWS trying to make both an id BOTH a friendly name and unique? Absolutely.

I don't think it's problematic to have the UUIDs. We'll never reference them, but it lets Prisma connect the models nicely and I'd rather have a real-life unique ID than not.